src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/AppointmentDtos.kt

```diff
@@ -117,3 +118,38 @@ data class AppointmentCreatedDto(
   val mainAppointmentId: Long,
   val recurringAppointments: Set<Long>? = null
 )
+
+fun VideoLinkBooking.preAppointmentDto() =
+  if (pre != null)
```

Comment:
meh... guess  you could use `.let` (or one of the other fancy methods ) to get rid of the if/else and the `!!`